applications/annotate_points.md

```diff
@@ -0,0 +1,314 @@
+# annotating videos with napari
+
+In this tutorial, we will use napari (requires version 0.3.2 or greater) to make a simple GUI application for annotating points in videos. This GUI could be useful for making annotations required to train algorithms for markless tracking of animals (e.g., [DeepLabCut](http://www.mousemotorlab.org/deeplabcut)). In this tutorial, we will cover creating and interacting with a Points layer with properties (i.e., labels for the points), connecting custom UI elements to events, and creating custom keybindings.
+
+At the end of this tutorial, we will have created a GUI for annotating points in videos that we can simply call by:
+
+```python
+im_path = '<path to directory with data>/*.png'
+output = '<path to directory with data>/annotations.csv'
+point_annotator(im_path, labels=['ear_l', 'ear_r', 'tail'], output_path=output)
+
+```
+that looks like this:
+
+![image]({{ '/assets/tutorials/point_annotator_demo.gif' | relative_url }})
+
+
+You can explore the project in [this repository](https://github.com/kevinyamauchi/PointAnnotator) or check out the main function below. We will walk through the code in the following sections.
+
+```python
+def point_annotator(
+        im_path: str,
+        labels: List[str],
+):
+    """Create a GUI for annotating points in a series of images.
+
+    Parameters
+    ----------
+    im_path : str
+        glob-like string for the images to be labeled.
+    labels : List[str]
+        list of the labels for each keypoint to be annotated (e.g., the body parts to be labeled).
+    """
+    stack = imread(im_path)
+    
+    with napari.gui_qt():
+        viewer = napari.view_image(stack, contrast_limits=[0, 256])
```

Comment:
is stack uint8? If so, you shouldn't need the `contrast_limits` anymore?

Reply:
Good point. Fixed